### ISSUE TITLE
"Crop out" feature request.

### ISSUE THREAD

**Issue by Alacritas1** [greenshot/greenshot#249]
Cropping out can be used to extract table rows with the heading line included. Right now i have to take 2 screenshots, one of the row and one of the heading, and merge them at destination. If I could crop out the rows above the target row with a tool I could export a single file with the heading and the target row. Total image size could get smaller automatically i imagine.
But I don't know how helpful this would be to others. Thanks in advance.

**Comment by kirkilj** [comment]
In the past, I used a tool that let you control this crop-out feature by supporting options to apply torn edges to the edges that were cropped out and to adjust the space injected between the cropped regions.

**Comment by tomgwilliams** [comment]
Agreed! This is one of the biggest features I'm missing from Greenshot compared to SnagIt. SnagIt has the Vertical or Horizontal cut out tool, which I use very very often. You basically set this up so that a user drags a full width/Ht Horizonal or Vertical region from start to end, and you splice the image into three different parts (if they run to the end of the image one of the 3 parts is just empty), deleting the center area, and shifting the two remaining splices together. 

As a follow on, it would be great to add a blurred, torn, etc. to the splice seams, but just a straight line will meet 95% of this ask. The other stuff is absolutely optimization.

**Comment by mischlrebl** [comment]
just a small side note: if the source is Excel, you could also just select the cells, copy them and paste it into GS.
would need two steps again, but should be a little bit faster.

**Comment by uwbwsvd** [comment]
I often need parts of the screen. The top and the bottom for example. I have to cut out the rest and shrink so the picture.

**Comment by Christian-Schulz** [comment]
> I often need parts of the screen. The top and the bottom for example. I have to cut out the rest and shrink so the picture.

This feature was already integrated with #388 in v1.3.244 , only this issue has not been closed.

<img width="854" height="1100" alt="Image" src="https://github.com/user-attachments/assets/552f476c-94ae-47b8-bfce-c797fb7fd96f" />

**Comment by uwbwsvd** [comment]
Ah. Thanks!
Is it possible to create a cutout with frayed edges?
So that you can tell something has been cut out there?

**Comment by Christian-Schulz** [comment]
> Is it possible to create a cutout with frayed edges?

Currently not. There is an open feature request #935 for that.
